extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -460,19 +452,24 @@ export class SchemaCompareDialog {
 	protected createSourceServerDropdown(): azdata.FormComponent {
 		this.sourceServerDropdown = this.view.modelBuilder.dropDown().withProperties(
 			{
-				editable: true,
+				editable: false,
```

Comment:
hmm why was it editable before? @kisantia any accessibility requirement here hat you remember?

Reply:
Not sure why was it editable before, but as of now, there seems to be no reason why it should stay editable- we should just be picking one server from the list.
@kisantia thoughts?

Reply:
it was editable for filtering to make it easier to find the server and database from the list. The Ok button isn't enabled if any dropdown has an invalid value(not from the dropdown's values)

Reply:
That makes sense. I think it makes sense to still keep it editable, thoughts?

Reply:
yeah let's keep it as editable